### PR TITLE
i18n(fr): update `guides/media/cloudinary.mdx`

### PR DESCRIPTION
**Pull request by ArmandPhilippot**
#### Description (required)

Adds changes from #9550 and #9659 to the French translation of `guides/media/cloudinary.mdx`

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/guides/media/cloudinary.mdx [hunk 1]
```diff
@@ -6,11 +6,221 @@ stub: true
...
+/>
+```
+
+Consultez la [documentation `<CldVideoPlayer>` de Cloudinary](https://astro.cloudinary.dev/cldvideoplayer/basic-usage) pour plus d'informations.
+
+## Activation des transferts vers Cloudinary
+
+Pour activer le transfert de fichiers dans l'interface utilisateur de votre site web ou de votre application, ajoutez le composant `<CldUploadWidget>` qui intégrera le [widget de transfert de Cloudinary](https://cloudinary.com/documentation/upload_widget).
+
+L'exemple suivant crée un widget pour autoriser les transferts non signés en transmettant un [préréglage de transfert] non signé(https://cloudinary.com/documentation/upload_presets):
```
Comment (thomasbnt) [inline src/content/docs/fr/guides/media/cloudinary.mdx:101]:
```suggestion
L'exemple suivant crée un widget pour autoriser les transferts non signés en transmettant un [préréglage de transfert](https://cloudinary.com/documentation/upload_presets) non signé :
```

**Review by thomasbnt — CHANGES_REQUESTED** [review]
Hello! Little typo link error, the rest is good!

**Review by thomasbnt — APPROVED** [review]
LGTM!

**Review by yanthomasdev — APPROVED** [review]
LGTM! Good work!

**Pull request merged**